apps/me/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
+import { environment } from '@env/environment';
+
+const routes: Routes = [
+  {
+    path: 'main',
```

Comment:
```suggestion
    path: 'home',
```
Name path home, better understand than main